.github/workflows/dispatch_release.yml

```diff
@@ -0,0 +1,31 @@
+### The auto release workflow triggered through dispatch request from CI
+name: auto-release
+
+# Run on workflow dispatch from CI
+on:
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        type: string
+        description: Name of the tag
+
+jobs:
+  auto-tag-and-release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Git User setup
+        run: "git config --local user.email Satellite-QE.satqe.com && git config --local user.name Satellite-QE"
+
+      - name: Tag latest commit
+        run: "git tag -a ${{ github.event.inputs.tag_name }} -m 'Tagged By SatelliteQE Automation User'"
+
+      - name: Push the tag to the upstream
+        run: "git push ${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}.git --tags"
+
+      - name: create a new release from the tag
+        env:
+          credentials: ${{ secrets.GH_TOKEN }}
+        run: "curl -L -X POST -H \"Authorization: Bearer ${{ secrets.SATQE_GH_TOKEN }}\" ${GITHUB_API_URL}/repos/${GITHUB_REPOSITORY}/releases -d '{\"tag_name\": \"${{ github.event.inputs.tag_name }}\", \"target_commitish\":\"master\", \"name\":\"${{ github.event.inputs.tag_name }}\", \"draft\":false, \"prerelease\":true, \"generate_release_notes\": true}'"
```

Comment:
Add a check here that the release was successful, if not add a slack notification.